apps/ledger-live-desktop/tools/config/renderer.esbuild.js

```diff
@@ -10,6 +10,18 @@ const {
 const { DOTENV_FILE } = require("../utils");
 const common = require("./common.esbuild");
 
+const nodeBuiltinsPlugin = {
+  name: "node-builtins",
+  setup(build) {
+    build.onResolve({ filter: /^node:/ }, args => {
+      return {
+        path: args.path.replace(/^node:/, ""),
+        external: true,
+      };
+    });
+  },
+};
+
```

Comment:
Do we not need @electron/rebuild as shown here https://www.electronjs.org/docs/latest/tutorial/using-native-node-modules ?

Also isn't it the same as the nodejs integration for the renderer thread for electron that is a security issue https://www.electronjs.org/docs/latest/tutorial/security#2-do-not-enable-nodejs-integration-for-remote-content ?

Reply:
Turns out that keeping `electron-store` at its current version still works and avoids the need for this: https://github.com/sindresorhus/electron-store/issues/213